### DISCUSSION TITLE
My resolutions do not match expected values.

### DISCUSSION THREAD

**Discussion by mdehrlich** [GerryDazoo/Slinger#156] [General]
I don't know if this me or a bigger issue.  I have an M2 and a 350.  both give me the same results.
Input resolutions to both slingboxs are 1920X1080 60i
Res 16 is great 1920X1080
Res 15 and 12 both give me 1280X540.  In VLC/Windows  it somehow knows to display in 16:9 format.  In OBS it displays stretched.
I would love to get 1280X720
Res 14 and 13 give me 320X240
ANY Help?

**Comment by GerryDazoo** [comment]
The only way I can get 1280x720 is to feed the box 1280x720.  There's some discussion at the end of this
https://github.com/GerryDazoo/Slinger/discussions/136. But I have the same deal 1080i in 1280x540 out.

**Reply by GerryDazoo** [reply]
PPS. In the video header there is a fields call DAR (Display Aspect Ratio) and SAR (source aspect ratio) some clients don't handle this  properly. But it's usually pretty straight forward to force a 16:9 output if needed.

**Comment by DoogieHowitzerWMD** [comment]
This has me thinking about my slingbox 500's now...

Years ago, I remember reading somewhere that the Slingbox 500 could only get 720p or 1080i at most with component, and you needed HDMI to get 1080p. Googling now, I can't find any confirmation of that, so I'm no longer sure.

Based on this logic, for the new Slinger server I'm running, I have the resolution set to 16 for a slingbox 500 I have that is connected with the HDMI connection, and the resolution set to 12 on another slingbox 500 that is using the component cables. **Now I'm wondering is there any reason I shouldn't be setting the resolution to 16 for both?**

From my streaming experience with Sling's servers, the stream from the one connected through HDMI always looked better to me. It's possible this was the result of some interference being picked up by the component cables (the ones I'm using are much longer than necessary and in a rat's nest of other cabling), or maybe it was just a placebo effect.

**Reply by GerryDazoo** [reply]
I don't think the 500 will eat 1080P but I haven't tested it myself.  My cable box has HDMI out and component out. So the HDMI is connected to my TV and component to the slingbox. If you're just streaming on your local network I see no reason not to use 1080i. But if I recall the refresh rate is only 30 Hz. So if it's sports you want 720P at 60 Hz is the sweet spot.

**Reply by DoogieHowitzerWMD** [reply]
I'll have to see what resolutions and frame rates are configurable from the output of my cable box

**Reply by sp222222** [reply]
they did an in depth study about ten years ago, HD cable vs component. This was before folks had 64" screens as common (before 4k also).
But basically, your eyes see the same thing but your ears might not unless using opitical with component.  HDMI is a conviencence thing in one cable vs components. 

the 500 won't do 1080p but it will do 720p at 60Hz (sports/live news)

**Comment by Pthepeg** [comment]
I did some quick tests on my 350 (component input) using VLC->tools to look at the stream info. 
I found there were only 3 resolutions available.
#define SBVS_320x240    1
#define SBVS_640x480    5
#define SBVS_1280x720    12
Any other resolution settings I tried, ended up falling back to one of those three resolutions, according to VLC

**Comment by mdehrlich** [comment]
I have a 350.  Resolution 16 works 1920X1080

**Reply by Pthepeg** [reply]
Hmmm that is different from mine. I have tested two 350s and using component input (video source = 1). VLC is on windows.
When I try resolution 16 I get 1280x720. 
The output from slingbox_server.py shows Resolution=16, and in VLC->Tools->Codec Information it shows Video Resolution: 1280x720.

**Reply by GerryDazoo** [reply]
The slingbox is pretty dumb. It will not upscale a video signal. I'm guessing here you are actually feeding the box 720P.
